templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.mm

```diff
@@ -83,6 +83,24 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull N
   return [super application:application continueUserActivity:userActivity restorationHandler:restorationHandler] || result;
 }
 
+// Leaves remote notification delegates for firebase ios sdk swizzling
```

Comment:
We need to think more about the comment here — for sure it's the case not only for Firebase, so more generic comment would fit better imho
cc @brentvatne